app/controllers/articles.controllers.js

```diff
@@ -23,3 +24,15 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.editArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+  changeArticleById(article_id, inc_votes)
+    .then((updated_article) => {
+      res.status(200).send({ updated_article });
+    })
+    .catch((err) => {
+      next(err);
```

Comment:
A nifty little shorthand you can use for the catch is: `.catch(next);` and this will invoke `next` with the _error_ if it's gone to the catch 👍